configuration/white_labels/base.py

```
@@ -28,16 +28,11 @@ def get_white_label(self) -> WhiteLabel:
-            {
-                "name": {"_default_message": "", "_label": ""},
-                "link": "",
-                "phone": {"_default_message": "", "_label": ""},
-            },
-        ]
-    }
+    # Resources shown on the Immediate Help tab (e.g., 2-1-1, state help lines). Empty by
+    # default rather than a placeholder entry: this now renders as a permanent tab (was
+    # previously only reachable via a button), so a placeholder with blank name/link would
+    # show an empty resource card to anyone on a white label that doesn't override this.
+    more_help_options = {"moreHelpOptions": []}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)